inst/stan/functions/delay_lpmf.stan

```diff
@@ -81,7 +81,7 @@ real delay_snap_lpmf(array[] int dummy, int start, int end, array[] int obs,
  * @param groups Array of group identifiers.
  * @param start Start index for the range of interest.
  * @param end End index for the range of interest.
- * @commonparams See Common Parameters for `delay_snap_lpmf` and `delay_group_lpmf`
+ * @see \ref common_parameters_page.
```

Comment:
This is good but ideally we would have them inherited if we can (if this is tricky perhaps we need a new issue)

Reply:
True. This was a lazy first pass but not ideal at all for navigation. I think we might be able to do it with [`copydoc`](https://www.doxygen.nl/manual/commands.html#cmdcopydoc). I'll look into it.